.github/ISSUE_TEMPLATE/--cart-checkout-feedback.md

```diff
@@ -0,0 +1,15 @@
+---
+name: "\U0001F4AC Feedback – Cart & Checkout Blocks"
+about: Submit feedback about the new block-based Cart & Checkout.
+labels: "type: support", "◼️ block: cart", "◼️ block: checkout"
```

Comment:
I'd like creating a new label for this `type: feedback` and use it for this template. It'll make it easier to search for just feedback related things.

Reply:
Good idea - added here 65386c4f88e90a1eeb0c2cf0d66aefbec0fdddc4 - though I haven't added the label itself, we can handle that separately I guess?

Also in that commit update the `what do you think is missing` prompt wording, and added `Cart & Checkout` to reinforce that we are looking for feedback on those blocks specifically.